global-script.lua

```diff
@@ -3088,4 +3148,87 @@ function tCompare(t1,t2)
     return table.concat(newTab,"|")
     end
     if cc2(t1) == cc2(t2) then return true else return false end
-end
\ No newline at end of file
+end
+
+
+function swapPlayerAreas(a, b)
+    local function tableSwap(table)
+        local temp = table[a]
+        table[a] = table[b]
+        table[b] = temp
+    end
+    local function tintSwap(table)
+        local oa = table[a]
+        local ob = table[b]
+        if type(oa) == "string" then
+            oa = getObjectFromGUID(oa)
+            ob = getObjectFromGUID(ob)
+        end
+        local ta = oa.getColorTint()
+        local tb = ob.getColorTint()
+        oa.setColorTint(tb)
+        ob.setColorTint(ta)
+        tableSwap(table)
+    end
+    local function positionSwap(table)
+        local oa = table[a]
+        local ob = table[b]
+        if type(oa) == "string" then
+            oa = getObjectFromGUID(oa)
+            ob = getObjectFromGUID(ob)
+        end
+        local ta = oa.getPosition()
+        local tb = ob.getPosition()
+        oa.setPosition(tb)
+        ob.setPosition(ta)
+    end
+
+    print("Player " .. a .. " is trading places with player " .. b .. ".")
```

Comment:
not sure if player needs to be specified in here, wouldn't "Red is trading places with Green" be good enough?